### PR TITLE
Sol cached database

### PR DESCRIPTION
**Pull request by Sosol02**

### CODE REVIEW

File: app/src/androidTest/java/com/github/onedirection/database/Item.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
     private final String s;
 
     // used by storer
-    private Item(Id id, int number, String s) {
+    Item(Id id, int number, String s) {
```
Comment (Flechman) [inline app/src/androidTest/java/com/github/onedirection/database/Item.java:15]: Shouldn't you just put public there in that case ?

**Review by Flechman — COMMENTED** [review]
The changes are almost nothing but refactoring so i guess things are kept clean.
Why did you create a MockDatabase class ?

**Comment by Sosol02** [comment]
The mocking database is to avoid creating a very big class in the last test of CachedDatabaseTests, and to allow code reuse in the future.

**Review by Flechman — APPROVED** [review]
Since the PR concerns mainly refactoring, there are no big concerns. Approved and thumbs up.

**Review by hugolan — APPROVED** [review]

**Pull request merged**